src/page-app/ClientSideRoutes.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * Important: This path is initially created in the gatsby-config.js file using the gatsby-plugin-create-client-paths.
+ * The pathname params are accessed because of the url fragment in that file.
+ */
+
+/* eslint-disable filenames/match-exported */
+import React from "react";
+import {
+  BrowserRouter as Router,
+  Redirect,
+  Route,
+  RouteComponentProps,
+  Switch,
+} from "react-router-dom";
```

Comment:
I don't remember if this is something we already talked about while you were developing this, but why aren't we using Reach Router here? could that be the source of your woes with handling URL paths?

Reply:
@macfarlandian great question. The reach/router is actually being phased out of Gatsby ( https://reacttraining.com/blog/reach-react-router-future/) but unrelated to that, the reach router can't handle multiple parameters in the URL. For example, /:scenarioId/:facilityId are not distinguishable.

To avoid writing a bunch of one-off helpers, react-router provides all of this.

Reply:
We may want/need @szhu to weigh in here. He was the one who originally moved us from react-router to reach-router as part of the Gatsby migration, and noted at the time that the two libraries were starting to converge.

Reply:
Hi, I'm so sorry, I must have missed this notification. I'm assuming the issue has been long resolved, but if not, feel free to contact me if there's something left to weigh in on.

Reply:
@szhu no worries! thanks for checking in but there's nothing more to resolve here for now.